ci/test_complete.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Compares options in `rg --help` output to options in zsh completion function
+
+set -e
+
+main() {
+    local rg="target/${TARGET}/release/rg"
```

Comment:
Oh hmm I see you're using the `--replace` flag. If you just change `rg` to `target/${TARGET}/debug/rg`, then I think that should be OK.